quupe-RN/js/screens/CategoryCameraGear/CategoryCameraGear.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
you do not need to create a screen for each category. You can have a single component showing the requested category items. Please make appropriate changes. If you are confused, come and see me.